src/text/raw_text_writer.rs

```diff
@@ -203,17 +238,21 @@ impl<W: Write> RawTextWriter<W> {
             if self.depth() > 0 {
                 // ...then this is the first value inside a container. We'll write the
                 // `space_after_container_start` so it will (e.g.) appear on its own line.
-                write!(&mut self.output, "{}", self.space_after_container_start)?;
+                write!(&mut self.output, "{}", self.whitespace_config.space_after_container_start)?;
             }
         } else {
             // Otherwise, this is not the first value in this container. Emit the container's
             // delimiter (for example: in a list, write a `,`) before we write the value itself.
             self.write_value_delimiter()?;
-            write!(&mut self.output, "{}", self.space_between_values)?;
```

Comment:
Did you consider hooking into the `write_value_delimiter` fn that's already switching on `ContainerType` or otherwise combining this with that logic?

Reply:
I did look at it, but I didn't see a satisfactory way. If I had to put this in `write_value_delimiter` then I'd still have to switch there on whether or not to add the `whitespace_config.space_between_values`. It also looks to me like `write_value_delimiter` is focused on the syntactic delimiters, not the presentation/whitespace additions. 

I'd rather leave all the "spacing" concerns in one place, I think.

This also makes me wonder whether the builder ought to validate that all the supplied whitespace values are semantically whitespace.